db/migrations/20230227082006_users.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE users(
+  id INT NOT NULL AUTO_INCREMENT,
+  kakao_id BIGINT(20) UNIQUE KEY NULL,
+  name VARCHAR(50) NOT NULL,
+  email VARCHAR(100) NOT NULL,
+  password VARCHAR(100) NOT NULL,
+  phone_number VARCHAR(100) NOT NULL UNIQUE KEY,
```

Comment:
phone_number는 카카오 로그인 때 못 받을 수도 있는 데이터이기 때문에 null을 허용해 주는 것이 좋습니다.
```suggestion
  phone_number VARCHAR(100) NULL UNIQUE KEY,
```